src/components/Header/SettingsToggle.tsx

```diff
@@ -48,6 +48,7 @@ const SettingsToggle = (props: SettingsToggleProps) => {
             onClick={onClick}
             key={title}
             ouiaId={rest.ouiaId ?? title}
+            data-ouia-component-id={rest.ouiaId ?? (title === 'User Access' ? 'user-access' : undefined)}
```

Comment:
We should extend the config of the dropdown items. The settings toggle is supposed to be generic.